### ISSUE TITLE
heroic 2.7.0 failed login to epic

### ISSUE THREAD

**Issue by danny2764** [Heroic-Games-Launcher/HeroicGamesLauncher#2662]
### Describe the bug

it seem failed login via heroic launcher for epic - seem to be captcha issues. please help.
Tested in steam os (steam deck) and portable windows ( Heroic-2.7.0-Portable-x64.exe ) exactly same issues.

### Add logs

```text
(11:41:17) INFO:    [Connection]:       Connectivity: check-online
(11:41:17) INFO:    [Connection]:       Pinging external endpoints
(11:41:17) INFO:    [Legendary]:        Running command: C:\Users\Danny\AppData\Local\Temp\2NxLVxjSZytygy5MQWIFhjpOfBl\resources\app.asar.unpacked\build\bin\win32\legendary --version
(11:41:17) INFO:    [Legendary]:        Legendary location: C:\Users\Danny\AppData\Local\Temp\2NxLVxjSZytygy5MQWIFhjpOfBl\resources\app.asar.unpacked\build\bin\win32\legendary
(11:41:17) INFO:    [Gog]:              GOGDL location: C:\Users\Danny\AppData\Local\Temp\2NxLVxjSZytygy5MQWIFhjpOfBl\resources\app.asar.unpacked\build\bin\win32\gogdl
(11:41:17) INFO:    [Gog]:              GOGDL location: C:\Users\Danny\AppData\Local\Temp\2NxLVxjSZytygy5MQWIFhjpOfBl\resources\app.asar.unpacked\build\bin\win32\gogdl
(11:41:18) WARNING: [Backend]:          Protocol already registered.
(11:41:18) ERROR:   [Backend]:          failed to update Error: ENOENT: no such file or directory, open 'C:\Users\Danny\AppData\Local\Temp\2NxLVxjSZytygy5MQWIFhjpOfBl\resources\app-update.yml'
(11:41:18) INFO:    [Connection]:       Connectivity: online
(11:41:18) INFO:    [Backend]:          User Not Found, removing it from Store
(11:41:20) INFO:    [Gog]:              Running command: C:\Users\Danny\AppData\Local\Temp\2NxLVxjSZytygy5MQWIFhjpOfBl\resources\app.asar.unpacked\build\bin\win32\gogdl --auth-config-path C:\Users\Danny\AppData\Roaming\heroic\gog_store\auth.json --version
(11:41:21) INFO:    [Backend]:          VCRuntime is installed
(11:41:22) INFO:    [Backend]:          Frontend Ready
(11:41:22) INFO:    [Backend]:          Loading Screen Ready
(11:41:22) WARNING: [Backend]:          Could not determine System Info
(11:41:22) INFO:    [Backend]:          Checking for new Heroic Updates
(11:41:22) INFO:    [Backend]:          Checking for current version changelog
(11:41:27) INFO:    [Backend]:          Starting the Download Queue
```

### Steps to reproduce

5.
![2023-04-29 11_46_49-Heroic Games Launcher](https://user-images.githubusercontent.com/91243936/235282190-b0339449-ac78-4b15-ad02-b2ddc81733b6.png)

### Expected behavior

expect can login to epic without issues.

### Screenshots

![2023-04-29 11_46_49-Heroic Games Launcher](https://user-images.githubusercontent.com/91243936/235282202-d9b61f27-9624-441c-81a8-6cd7ad86c947.png)

### Heroic Version

Latest Stable

### System Information

- OS [e. g. "Steam deck OS"-2.7.0]:
- OS [e. g. " Heroic-2.7.0-Portable-x64.exe "

**Comment by danny2764** [comment]
manage to login now with alternative login

**Comment by Nocccer** [comment]
Already fixed in main branch.

**Comment by SeongGino** [comment]
Apparently not. Tried to claim a game using 2.7.1 which gave me:
![2023_05-04 Heroic Games Launcher-014617](https://user-images.githubusercontent.com/7321839/236122379-8ba56c51-962c-4c76-8bdd-3143e7e30e6b.png)

Tried clearing out the heroic config folder and started fresh (minus config.json), got prompted to re-login, and now I can't. 
![2023_05-04 Heroic Games Launcher-015137](https://user-images.githubusercontent.com/7321839/236122642-6296f92f-af46-4e16-a89a-f71769369535.png)


```
(01:49:36) INFO:    [Legendary]:        Refreshing library...
(01:49:36) INFO:    [Legendary]:        Refreshing Epic Games...
(01:49:36) INFO:    [Connection]:       Connectivity: check-online
(01:49:36) INFO:    [Connection]:       Pinging external endpoints
(01:49:36) INFO:    [Legendary]:        Running command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(01:49:36) INFO:    [Legendary]:        Legendary location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary
(01:49:36) INFO:    [Gog]:              GOGDL location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(01:49:36) INFO:    [Gog]:              GOGDL location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(01:49:36) INFO:    [Legendary]:        Game list updated, got 242 games & DLCs
(01:49:36) WARNING: [Backend]:          Protocol already registered.
(01:49:36) INFO:    [Gog]:              Running command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/[...]/.config/heroic/gog_store/auth.json --version
(01:49:36) INFO:    [Connection]:       Connectivity: online
(01:49:36) INFO:    [Legendary]:        Running command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(01:49:36) INFO:    [Backend]:          AreWeAntiCheatYet data downloaded
(01:49:36) INFO:    [Backend]:          Loading Screen Ready
(01:49:37) INFO:    [Frontend]:         Refreshing Library
(01:49:37) WARNING: [Backend]:          listUpdateableGames not implemented on Sideload Library Manager
(01:49:37) INFO:    [Gog]:              Found 0 game(s) to update
(01:49:37) INFO:    [Backend]:          Frontend Ready
(01:49:37) INFO:    [Backend]:          Checking for new Heroic Updates
(01:49:37) INFO:    [Backend]:          

Heroic Version: 2.7.1 Eustass Kid
Legendary Version:  0.20.32 Dark Energy (hotfix #6)
GOGdl Version: 0.7.1

Electron Version: 23.1.4
Chrome Version: 110.0.5481.192
NodeJS Version: 18.12.1

OS: EndeavourOS KERNEL: 6.3.0-273-tkg-cfs ARCH: x64
CPU: AMD Ryzen 5 5600X 6-Core Processor @3.7 
RAM: Total: 15.53 GiB Available: 8.64 GiB
GRAPHICS: GPU0: NVIDIA GeForce RTX 3060 Ti VRAM: 7971MB DRIVER: 530.41.03
PROTOCOL: x11
```
